client/web/src/components/widgets/inputs/MenuBarInput.vue

```diff
@@ -42,13 +49,108 @@
 import { defineComponent } from "vue";
 import Icon from "../labels/Icon.vue";
 import { ApplicationPlatform } from "../../window/MainWindow.vue";
+import MenuList, { MenuListEntry, MenuListEntries } from "../floating-menus/MenuList.vue";
+import { MenuDirection } from "../floating-menus/FloatingMenu.vue";
+
+const wasm = import("../../../../wasm/pkg");
+
+const menuEntries: MenuListEntries = [
+	{
+		icon: "GraphiteLogo",
+		ref: undefined,
+		children: [[{ label: "Visit project GitHub…", action: () => window.open("https://github.com/GraphiteEditor/Graphite", "_blank") }]],
+	},
+	{
+		label: "File",
+		ref: undefined,
+		children: [
+			[
+				{ label: "New", icon: "FileNew", shortcut: ["Ctrl", "N"] },
+				{ label: "Open…", shortcut: ["Ctrl", "O"] },
+				{
+					label: "Open Recent",
+					shortcut: ["Ctrl", "⇧", "O"],
+					children: [
+						[{ label: "Reopen Last Closed", shortcut: ["Ctrl", "⇧", "T"] }, { label: "Clear Recently Opened" }],
+						[
+							{ label: "Some Recent File.gdd" },
+							{ label: "Another Recent File.gdd" },
+							{ label: "An Older File.gdd" },
+							{ label: "Some Other Older File.gdd" },
+							{ label: "Yet Another Older File.gdd" },
+						],
+					],
+				},
+			],
+			[
+				{ label: "Close", shortcut: ["Ctrl", "W"] },
```

Comment:
Wont this just close the browser window?

Reply:
Yes, a lot of these aren't supported because the browser doesn't let you override them. We might have to use the keyboard lock API.